# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dvdisaster/parallelism.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./locale/create-makefile.orig	2009-11-15 23:19:02.000000000 +0100
+++ ./locale/create-makefile	2009-11-15 23:20:50.000000000 +0100
@@ -30,15 +30,15 @@
 
 # Create the locale files. 
 
+messages.pot:
+	@cd ..; xgettext --no-wrap -cTRANSLATORS: -o locale/messages.pot -k_ -k_utf \$(CFILES_NP)
 EOF
 
 for i in $prefixes; do
 cat >> Makefile <<EOF
-$i.po: \$(CFILES)
+$i.po: messages.pot \$(CFILES)
 	@echo "Updating $i.po"
-	@cd ..; xgettext --no-wrap -cTRANSLATORS: -o locale/messages.pot -k_ -k_utf \$(CFILES_NP)
 	@msgmerge -q -U --no-wrap --no-fuzzy-matching $i.po messages.pot
-	@rm -f messages.pot
 
 $i/LC_MESSAGES/dvdisaster.mo: $i.po
 	@echo "Updating $i.mo"
@@ -46,4 +46,4 @@
 
 
 EOF
-done
\ No newline at end of file
+done
